fiftyone/core/odm/mixins.py

```diff
@@ -715,7 +715,9 @@ def _extract_extra_updates(self, update_doc, filtered_fields):
             for d in update_doc.values():
                 for k in d.keys():
                     for ff in filtered_fields:
-                        if k.startswith(ff) and not k.lstrip(ff).count("."):
+                        if k.startswith(ff) and not k.replace(
```

Comment:
nit: prefer `k[len(ff):]` instead of `k.replace(ff, "", 1)`.

Same thing below.